css/css-break/widows-orphans-022.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Fragmentation level 3 Test: 'orphans', 'widows' and content distribution in columns</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css-break-3/#widows-orphans">
+  <link rel="match" href="reference/widows-orphans-022-ref.html">
+
+  <!--
+  Date created: November 26th 2020
+  Last modified: November 27th 2020
+  -->
+
+  <!--
+
+  Bug 137367: Implement orphans and widows
+  https://bugzilla.mozilla.org/show_bug.cgi?id=137367
+
+  -->
+
+  <meta name="flags" content="">
+  <meta name="assert" content="When column boxes are filled sequentially, their content should be distributed and fragmented in accordance with the 'orphans' and the 'widows' declarations. In the test, since the 3rd column box was going to get only 1 line box, then a class B break point should occur between the '8' and the '9' so that there is a minimum of 2 line boxes at the top of the 3rd column box.">
+
+  <!--
+
+    Breaking at a class B break point is allowed only if (...)
+    the number of line boxes between the break and the end
+    of the box is the value of widows or more.
+    4.4 Unforced Breaks
+    https://www.w3.org/TR/css-break-3/#unforced-breaks
+
+  -->
+
+  <!--
+
+  height: auto and orphans = 1 and widows = 2: widows-orphans-022.html
+
+  height: auto and orphans = 1 and widows = 3: widows-orphans-023.html
+
+  height: auto and orphans = 2 and widows = 2: widows-orphans-024.html
+
+  height: auto and orphans = 2 and widows = 3: widows-orphans-025.html
+
+  height: auto and orphans = 2 and widows = 6: widows-orphans-026.html
+
+  height: auto and orphans = 3 and widows = 7: widows-orphans-027.html
+
+  height: auto and orphans = 6 and widows = 4: widows-orphans-028.html
+
+  height: auto and orphans = 7 and widows = 3: widows-orphans-029.html
+
+
+
+  height: 104px and orphans = 1 and widows = 2: widows-orphans-032.html
+
+  height: 104px and orphans = 1 and widows = 3: widows-orphans-033.html
+
+  height: 104px and orphans = 2 and widows = 2: widows-orphans-034.html
+
+  height: 104px and orphans = 2 and widows = 3: widows-orphans-035.html
+
+  height: 156px and orphans = 2 and widows = 6: widows-orphans-036.html
+
+  height: 208px and orphans = 3 and widows = 7: widows-orphans-037.html
+
+  height: 234px and orphans = 6 and widows = 4: widows-orphans-038.html
+
+  height: 234px and orphans = 7 and widows = 3: widows-orphans-039.html
+
+  -->
+
+  <style>
+  div
+    {
+      border: orange solid 4px;
+      font-size: 20px;
+      height: auto;
```

Comment:
What's this supposed to test? height:auto combined with column-fill:auto means no soft breaks at all (only forced ones). I believe Gecko gets this right, while Blink still follows an older version of the spec, which forces column balancing in this case (which is behavior I suppose we shouldn't test for).

Reply:
I will filename-rename the tests to start counting from 006 instead of 022.
I will remove the comment describing the serie of tests.
I will remove the comment about the Class B break point.
I will remove the comment providing the widows definition.
I will remove the 'height: auto' serie of tests and only keep 
one test with 'height: auto' with the correct rendering.
I am planning to do all this in the next 2-3 days.